vizro-core/src/vizro/static/css/layout.css

```diff
@@ -175,3 +176,24 @@ div.dashboard_container .custom-tooltip {
 #page-container div[hidden] {
   display: none;
 }
+
+.collapsable-div {
+  background-color: var(--surfaces-bg-02);
+  height: 100%;
+  padding-top: 44px;
+  width: 12px;
+  padding-right: 12px;
+}
+
+#collapse_icon.material-symbols-outlined {
+  z-index: 1;
+  position: absolute;
+  background-color: var(--surfaces-bg-card);
```

Comment:
I think we've discussed w/ Joe that we would update the background-color and color here?

The background-color is currently not optimal, because on dark / light it looks different e.g. in the dark theme it has a different color to the side panel, while on the light theme it looks like it has exactly the same color?

I think for color we wanted to use the disabled color instead of text-secondary